<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="grafoviWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>KONCENTRACIJE</string>
       </attribute>
       <layout class="QVBoxLayout" name="koncPanelLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="zsWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>ZERO / SPAN</string>
       </attribute>
       <layout class="QVBoxLayout" name="zsPanelLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1159</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_REST">
    <property name="title">
     <string>&amp;REST</string>
    </property>
    <addaction name="action_log_in"/>
    <addaction name="action_log_out"/>
    <addaction name="separator"/>
    <addaction name="action_reconnect"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menu_Satni_graf">
     <property name="title">
      <string>&amp;Satni graf</string>
     </property>
     <addaction name="action_satni_grid"/>
     <addaction name="action_satni_cursor"/>
     <addaction name="action_satni_minor_ticks"/>
     <addaction name="action_satni_legend"/>
    </widget>
    <widget class="QMenu" name="menu_Minutni_graf">
     <property name="title">
      <string>&amp;Minutni graf</string>
     </property>
     <addaction name="action_minutni_grid"/>
     <addaction name="action_minutni_cursor"/>
     <addaction name="action_minutni_minor_ticks"/>
     <addaction name="action_minutni_legend"/>
    </widget>
    <widget class="QMenu" name="menu_SPAN_graf">
     <property name="title">
      <string>&amp;SPAN graf</string>
     </property>
     <addaction name="action_SPAN_legend"/>
    </widget>
    <widget class="QMenu" name="menu_ZERO_graf">
     <property name="title">
      <string>&amp;ZERO graf</string>
     </property>
     <addaction name="action_ZERO_legend"/>
    </widget>
    <addaction name="menu_Satni_graf"/>
    <addaction name="menu_Minutni_graf"/>
    <addaction name="menu_ZERO_graf"/>
    <addaction name="menu_SPAN_graf"/>
    <addaction name="separator"/>
    <addaction name="action_zoom"/>
    <addaction name="action_zoom_out"/>
    <addaction name="separator"/>
    <addaction name="action_stil_grafova"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_REST"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Kontrolni prozor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_log_in"/>
   <addaction name="action_log_out"/>
   <addaction name="action_reconnect"/>
   <addaction name="separator"/>
   <addaction name="action_stil_grafova"/>
   <addaction name="separator"/>
   <addaction name="action_zoom"/>
   <addaction name="action_zoom_out"/>
  </widget>
  <action name="action_log_in">
   <property name="text">
    <string>&amp;Log in</string>
   </property>
  </action>
  <action name="action_log_out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Log out</string>
   </property>
  </action>
  <action name="action_reconnect">
   <property name="text">
    <string>&amp;Reconnect</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_satni_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Grid</string>
   </property>
   <property name="toolTip">
    <string>Toggle grid na grafu satno agregiranih podataka</string>
   </property>
  </action>
  <action name="action_satni_cursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Cursor</string>
   </property>
   <property name="toolTip">
    <string>Toggle cursor assist na grafu sa satno agregiranim podacima.</string>
   </property>
  </action>
  <action name="action_satni_minor_ticks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Minor ticks</string>
   </property>
   <property name="toolTip">
    <string>Toggle minor tickove na grafu sa satno agregiranim podacima.</string>
   </property>
  </action>
  <action name="action_satni_legend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Legend</string>
   </property>
   <property name="toolTip">
    <string>Toggle prikaz legende na grafu sa satno agregiranim podacima.</string>
   </property>
  </action>
  <action name="action_minutni_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Grid</string>
   </property>
   <property name="toolTip">
    <string>Toggle grid na grafu sa minutnim podacima.</string>
   </property>
  </action>
  <action name="action_minutni_cursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Cursor</string>
   </property>
   <property name="toolTip">
    <string>Toggle cursor assist na grafu sa minutnim podacima.</string>
   </property>
  </action>
  <action name="action_minutni_minor_ticks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Minor ticks</string>
   </property>
   <property name="toolTip">
    <string>Toggle minor ticks na grafu sa minutnim podacima.</string>
   </property>
  </action>
  <action name="action_minutni_legend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Legend</string>
   </property>
   <property name="toolTip">
    <string>Toggle prikaz legende na grafu sa minutnim podacima.</string>
   </property>
  </action>
  <action name="action_zoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Toggle zoom mod za grafove</string>
   </property>
  </action>
  <action name="action_zoom_out">
   <property name="text">
    <string>&amp;Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out sve grafove</string>
   </property>
  </action>
  <action name="action_stil_grafova">
   <property name="text">
    <string>&amp;Stil grafova</string>
   </property>
   <property name="toolTip">
    <string>Promjena stila grafova.</string>
   </property>
  </action>
  <action name="action_ZERO_legend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Legend</string>
   </property>
   <property name="toolTip">
    <string>Toggle legende za ZERO graf</string>
   </property>
  </action>
  <action name="action_SPAN_legend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Legend</string>
   </property>
   <property name="toolTip">
    <string>Toggle legende za SPAN graf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
